nuxeo-core/nuxeo-core-storage-dbs/src/main/java/org/nuxeo/ecm/core/storage/dbs/DBSDocument.java

```diff
@@ -351,7 +351,7 @@ public Document getChild(String name) {
 
     @Override
     public List<String> getChildrenIds() {
-        return session.getChildrenIds(id);
+        return session.getAllChildrenIds(id);
```

Comment:
Ok now that I reread the final code I think we can keep the old name `getChildrenIds`, as the new method with the "exclude" args is distinct enough. Sorry about changing opinion several time.
Also applies to other `getAll*` methods we renamed...

Reply:
No problem, hopefully IDE help a lot with refactoring !

Reply:
I did the same with 
`DBSTransactionState#getAllChildrenStates`
`DBSTransactionState#getAllChildrenIds`